<ui version="4.0" >
 <class>qdOptions</class>
 <widget class="QDialog" name="qdOptions" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Options</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>150</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTreeWidget" name="qtwChoices" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QStackedWidget" name="qswChoices" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>0</y>
      <width>401</width>
      <height>131</height>
     </rect>
    </property>
    <property name="currentIndex" >
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_2" />
    <widget class="QWidget" name="page" >
     <widget class="QPushButton" name="qpbAddContentPlugin" >
      <property name="geometry" >
       <rect>
        <x>240</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Add</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="qtwContentPlugins" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>10</y>
        <width>401</width>
        <height>80</height>
       </rect>
      </property>
      <property name="rootIsDecorated" >
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="qpbRemoveContentPlugin" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>320</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Remove</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>qdOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>qdOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
